Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -539,7 +539,7 @@ bool WebProcessProxy::shouldSendPendingMessage(const PendingMessage& message)
     if (message.encoder->messageName() == IPC::MessageName::WebPage_LoadRequestWaitingForProcessLaunch) {
         auto buffer = message.encoder->buffer();
         auto bufferSize = message.encoder->bufferSize();
```

Comment:
It may be nice for `message.encoder` to return us the buffer as a std::span directly.

Reply:
This patch already snowballed a bit.  If I change that, I suspect this patch will get even bigger.

I think it will be obvious when changing `IPC::Encoder` to use `std::span` to make this change then.

Reply:
All I suggested was introducing a new member function that returns a span, not modify any existing member function. Therefore, I don't understand the snowball risk. It only makes the code you change nicer.